server/node.go

```diff
@@ -577,8 +577,9 @@ func (n *Node) connectGossip() {
 
 // startGossip loops on a periodic ticker to gossip node-related
 // information. Starts a goroutine to loop until the node is closed.
-func (n *Node) startGossip(ctx context.Context, stopper *stop.Stopper) {
+func (n *Node) startGossip(stopper *stop.Stopper) {
 	stopper.RunWorker(func() {
```

Comment:
Ditto what @tamird said. Seems like `RunWorker` should get the same context treatment as `RunAsyncTask`.